app/templates/buyers/review_brief.html

```diff
@@ -0,0 +1,91 @@
+{% extends "_base_page.html" %}
+{% from "components/tabs/macro.njk" import govukTabs %}
```

Comment:
This might be better positioned in the `_base_page.html` so we know which components we are importing in one place and it might be used else where.

Reply:
I agree! Can you tell this branch was started before this repo had the new FE stuff pulled in? 😹